pkg/parca/parca.go

```diff
@@ -115,6 +115,8 @@ type Flags struct {
 	ExternalLabel      map[string]string `kong:"help='Label(s) to attach to all profiles in scraper-only mode.'"`
 
 	ExperimentalArrow bool `default:"false" help:"EXPERIMENTAL: Enables Arrow ingestion, this will reduce CPU usage but will increase memory usage."`
+
+	Hidden FlagsHidden `embed:"" prefix:"" hidden:""`
```

Comment:
```suggestion
	Hidden FlagsHidden `embed:"" prefix:""`
```
"hidden" doesn't work with the embed. This would be my bad if I left it in Parca Agent.

Reply:
Good catch 👍